### PR TITLE
Update Config

### PR DESCRIPTION
**Pull request by nasbench**
- Added more double extensions
- Added basic logging to image load at least from "public folder" instead of nothing
- Removed the "dllhost ProcessID" command line exclusion. As this is important to have for COM hijacking via dllhost

### CODE REVIEW

File: sysmonconfig-export-block.xml [hunk 1]
```diff
@@ -1322,6 +1323,7 @@
 			<!-- Executables dropped to suspicious folders -->
 			<TargetFilename condition="begin with">C:\Users\Public\</TargetFilename> <!-- often used staging directories; could cause false positives --> 
 			<TargetFilename condition="begin with">C:\Perflogs\</TargetFilename> <!-- often used staging directories --> 
+			<TargetFilename condition="begin with">C:\AppData\LocalLow\</TargetFilename> <!-- often used staging directories -->
```
Comment (Neo23x0) [inline sysmonconfig-export-block.xml]: This looks wrong

Comment (nasbench) [reply sysmonconfig-export-block.xml]: You're right It's 100% wrong. Should be fixed now

File: sysmonconfig-export-block.xml [hunk 2]
```diff
@@ -1322,6 +1323,7 @@
 			<!-- Executables dropped to suspicious folders -->
 			<TargetFilename condition="begin with">C:\Users\Public\</TargetFilename> <!-- often used staging directories; could cause false positives --> 
 			<TargetFilename condition="begin with">C:\Perflogs\</TargetFilename> <!-- often used staging directories --> 
+			<TargetFilename condition="begin with">C:\Users\;\AppData\LocalLow\</TargetFilename> <!-- often used staging directories -->
```
Comment (Neo23x0) [inline sysmonconfig-export-block.xml]: Isn't LocalLow used by browser to store data downloaded from the Internet, e.g. Internet Explorer

Comment (Neo23x0) [reply sysmonconfig-export-block.xml]: I wouldn't want to interfere with user downloads that are legitimate

Comment (nasbench) [reply sysmonconfig-export-block.xml]: It shouldn't contain any exe to my knowledge as those should be dropped to local/temp but I could be wrong. I'll remove it then by precaution

**Pull request merged**